src/python/bot/fuzzers/syzkaller/runner.py

```diff
@@ -87,7 +86,7 @@ def repro(self, repro_timeout, repro_args):
     result = self.run_and_wait(additional_args, timeout=repro_timeout)
     logs.log('Syzkaller testcase stopped.')
     return engine.ReproduceResult(result.command, result.return_code,
-                                  result.time_executed, str(result.output))
+                                  result.time_executed, result.output)
```

Comment:
Similar to below, this should be wrong, as str shouldn't be called on a byte output. Do you remember where it failed ?